postman/AskDarcel%20API.postman_collection.json

```diff
@@ -517,7 +517,7 @@
 				"method": "GET",
 				"header": [],
 				"url": {
-					"raw": "{{base_url}}/services/search?site_id=1",
+					"raw": "{{base_url}}/services?site_id=1",
```

Comment:
Why are we removing the /search suffix here from the URL? This test is intended to test the algolia endpoint which uses the /search suffix right?